From 61facbd9c0777a6764453f297f8beefda8864f93 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Wed, 27 Mar 2013 11:52:10 +0100
Subject: [PATCH] Start daemon on different upstart event and not in
 dependency of connman

As of right now there isn't a upstart job for connman and it's still started through the
sysvinit compatibility layer. webos-connman-adapter itself is feasible of detecting a
started connman daemon through dbus signaling so it doesn't need to be started before.

Open-webOS-DCO-1.0-Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 files/launch/webos-connman-adapter.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/launch/webos-connman-adapter.in b/files/launch/webos-connman-adapter.in
index b63ce2c..a17f1b0 100644
--- a/files/launch/webos-connman-adapter.in
+++ b/files/launch/webos-connman-adapter.in
@@ -18,7 +18,7 @@
 
 # -*- mode: conf; -*-
 
-start on stopped connman
+start on stopped finish
 
 respawn
 exec @WEBOS_INSTALL_SBINDIR@/webos-connman-adapter
-- 
1.7.10.4

